zerver/lib/hotspots.py

```diff
@@ -75,26 +79,46 @@ def get_next_hotspots(user: UserProfile) -> List[Dict[str, object]]:
     if not settings.TUTORIAL_ENABLED:
         return []
 
-    if user.tutorial_status == UserProfile.TUTORIAL_FINISHED:
-        return []
-
     seen_hotspots = frozenset(
         UserHotspot.objects.filter(user=user).values_list("hotspot", flat=True)
     )
-    for hotspot in INTRO_HOTSPOTS.keys():
-        if hotspot not in seen_hotspots:
-            return [
-                {
-                    "name": hotspot,
-                    "title": str(INTRO_HOTSPOTS[hotspot]["title"]),
-                    "description": str(INTRO_HOTSPOTS[hotspot]["description"]),
-                    "delay": 0.5,
-                }
-            ]
+
+    hotspots = []
+
+    for name, base_hotspot in NON_INTRO_HOTSPOTS.items():
+        if name in seen_hotspots:
+            continue
+
+        hotspot = {
+            **base_hotspot,
+            "name": name,
+            "title": str(base_hotspot["title"]),
+            "description": str(base_hotspot["description"]),
+            "is_non_intro_hotspot": True,
+        }
+        hotspots.append(hotspot)
+
+    if user.tutorial_status == UserProfile.TUTORIAL_FINISHED:
+        return hotspots
+
+    for name, base_hotspot in INTRO_HOTSPOTS.items():
+        if name in seen_hotspots:
+            continue
+
+        # Make a copy to set delay and finalize i18n strings.
+        hotspot = {
+            **base_hotspot,
+            "name": name,
+            "title": str(base_hotspot["title"]),
+            "description": str(base_hotspot["description"]),
+            "delay": 0.5,
+        }
+        hotspots.append(hotspot)
+        return hotspots
 
     user.tutorial_status = UserProfile.TUTORIAL_FINISHED
     user.save(update_fields=["tutorial_status"])
-    return []
+    return hotspots
```

Comment:
I suppose we could avoid the extra query if we changed this to only set `UserProfile.TUTORIAL_FINISHED` if `hotspots == []` at this juncture.  

@alya we should discuss the product question for whether we want this new drafts hotspot to appear only for "new users" or for existing users; that's the product question that determines how this code should read.